CONTRIBUTING.md

```diff
@@ -14,21 +16,60 @@
   - [Creating Issues for New Components](#creating-issues-for-new-components)
   - [Contributing Components](#contributing-components)
   - [Adding Styling for your Components](#adding-styling-for-your-components)
-  - [Using Generators](#using-generators)
+
 - [Guidelines and Requirements](#guidelines-and-requirements)
   - [React Component Requirements](#react-component-requirements)
   - [Code Consistency](#code-consistency)
   - [Code Contribution Guidelines](#code-contribution-guidelines)
+- [Troubleshooting](#troubleshooting)
 - [Becoming a Maintainer](#becoming-a-maintainer)
   - [How do I become a maintainer?](#how-do-i-become-a-maintainer)
   - [How do I lose maintainers status?](#how-do-i-lose-maintainers-status)
   - [Quick Tips for New Maintainers](#quick-tips-for-new-maintainers)
 
+## Quick Start for New Contributors
+
+New to contributing to PatternFly React? Here's how to get started quickly:
+
+1. **🍴 Fork and Clone**: Fork the repository and clone it locally
+2. **📦 Install Dependencies**: Run `yarn install` to install dependencies  
+3. **🏗️ Build**: Run `yarn build` to build the project
+4. **🔍 Find an Issue**: Look for issues labeled [`good first issue`](https://github.com/patternfly/patternfly-react/labels/good%20first%20issue)
+5. **🌿 Create a Branch**: Create a new branch for your changes
+6. **✅ Test**: Run `yarn test` to ensure tests pass
+7. **📝 Submit PR**: Create a pull request with a clear description
+
+**Need help?** Join us on [PatternFly Slack](https://patternfly.slack.com/) in the `#patternfly-react` channel!
+
+## Community Contributors Hall of Fame
+
+We want to recognize and celebrate our amazing community contributors who have made significant contributions to PatternFly React in the past year! 🎉
+
+### Top Community Contributors (Last 12 Months as on July 1, 2025)
+
+The following contributors (excluding PatternFly team members and bots) have made outstanding contributions to the project:
+
+1. **@Mash707** - 49 contributions
+2. **@adamviktora** - 10 contributions
+3. **@logonoff** - 5 contributions
+
+### Notable Contributors
+
+Thank you to all community contributors who have helped improve PatternFly React! Every contribution makes a difference.
+
+Thank you to all our community contributors for helping make PatternFly React better! Every contribution, whether it's code, documentation, bug reports, or feature requests, helps improve the library for everyone.
```

Comment:
This paragraph is doubled and this section seems to be a dupe of the above `Community Contributors Hall of Fame`.